lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -799,6 +800,24 @@ FunctionType *PatchEntryPointMutate::generateEntryPointType(uint64_t *inRegMask)
     entryArgIdxs.vs.instanceId = argTys.size();
     argTys.push_back(Type::getInt32Ty(*m_context)); // Instance ID
 
+    // In relocatable shader, the vertex inputs are passed to the vertex shader
+    // in VGPRS.  Add them to the interface.
+    if (m_pipelineState->getLgcContext()->buildingRelocatableElf()) {
+      // TODO: See if we can store the vertex input descriptions on sorted order based on location.  It will make
+      // finding a particular input faster.
```

Comment:
I sort the list below.  That does not seem to cause problem, but it might be better if there is guarantee that they are always sorted.  What are your thoughts on that?  Is it already true and I didn't know?

Reply:
This seems to have disappeared? I really don't like the fact that Github won't show me the diff anymore :/

Reply:
Yes, I changed that part of the code to no reference the vertex input descriptions.  So it did become obsolete.  Too bad did that after I made that comment.